addons/rivet/devtools/rivet_cli_output.gd

```diff
@@ -46,4 +46,5 @@ func _parse_output(internal_output: Array) -> void:
 		self.output = json.data
 	else:
 		# If the output is not JSON, throw an error
-		print("Error parsing JSON output: " + str(error))
+		RivetPluginBridge.error("Invalid response from the command line tool. Please report this issue and try again.")
```

Comment:
@NathanFlurry will the CLI report any error to Sentry? If so, should this instead be "this issue has been reported to the developers"?